glide.lock

```diff
@@ -1,31 +1,37 @@
-hash: 5df50308c509024912fc12bda5f9b15eb2661540f64b719707a403a2e52f659a
-updated: 2017-08-02T12:48:17.628000263-07:00
+hash: 7ab8c3bc47c8889cc1ba5af07a896c6ba2aad7889b79c04f931a644b7f2a1a87
+updated: 2017-08-10T11:40:04.338746051-04:00
 imports:
+- name: github.com/alecthomas/template
+  version: a0175ee3bccc567396460bf5acd36800cb10c49c
+  subpackages:
```

Comment:
Can you give me a list of all new and upgraded dependencies? I'll make sure they're registered with MSFT OSS.